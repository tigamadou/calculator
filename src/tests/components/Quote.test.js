import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quotes from '../../components/Quote';

describe('Quotes', () => {
  test('renders Quotes component ', () => {
    render(<Quotes />);
    // eslint-disable-next-line no-unused-expressions
    expect(screen.getByText('William Paul Thurston')).toBeInTheDocument;
  });
  test('should match with snapshot', () => {
    const tree = renderer
      .create(<Quotes />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
