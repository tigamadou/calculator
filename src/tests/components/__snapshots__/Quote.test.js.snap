// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes should match with snapshot 1`] = `
<div
  className="card quote"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
  </p>
  <p
    className="text-primary"
  >
    William Paul Thurston
  </p>
</div>
`;
